xonsh/vox.py

```diff
@@ -0,0 +1,197 @@
+from os.path import join, basename, exists
+import os
+from shutil import rmtree
+import venv
+
+import builtins
+
+
+class Vox:
+    """Vox is a virtual environment manager for xonsh."""
+
+    def __init__(self):
+        """Ensure that $VIRTUALENV_HOME is defined and declare the available vox commands"""
+
+        if not builtins.__xonsh_env__.get('VIRTUALENV_HOME'):
+            if os.name == 'nt':
+                home_path = builtins.__xonsh_env__['USERPROFILE']
+
+            elif os.name == 'posix':
+                home_path = builtins.__xonsh_env__['HOME']
```

Comment:
Fixed in https://github.com/moigagoo/xonsh/commit/7488142dbc55d4cb10138394e420fee2a4c069d8